catalogue/webapp/components/SearchPageLayout/SearchPageLayout.tsx

```diff
@@ -0,0 +1,259 @@
+  const currentSearchCategory =
+    router.pathname === '/search'
+      ? 'overview'
+      : router.pathname.slice(router.pathname.lastIndexOf('/') + 1);
+
+  const basePageMetadata = {
+    openGraphType: 'website',
+    siteSection: 'collections',
+    jsonLd: { '@type': 'WebPage' },
+    hideNewsletterPromo: true,
+  } as const;
+
+  const defaultPageLayoutMetadata = {
+    ...basePageMetadata,
+    title: 'Search Page',
+    description: 'TBC',
+    url: { pathname: '/search', query: {} },
+  };
+
+  const [pageLayoutMetadata, setPageLayoutMetadata] = useState(
+    defaultPageLayoutMetadata
+  );
+  useEffect(() => {
+    const { query } = router.query;
+    switch (currentSearchCategory) {
+      case 'overview':
+        setPageLayoutMetadata(defaultPageLayoutMetadata);
+        break;
+      case 'exhibitions':
+        setPageLayoutMetadata({
+          ...basePageMetadata,
+          description: 'copy pending',
+          title: `${query ? `${query} | ` : ''}Exhibition Search`,
+          url: { pathname: '/search/exhibitions', query: router.query },
+        });
+        break;
+      case 'events':
+        setPageLayoutMetadata({
+          ...basePageMetadata,
+          description: 'copy pending',
+          title: `${query ? `${query} | ` : ''}Events Search`,
+          url: { pathname: '/search/events', query: router.query },
+        });
+        break;
+      case 'stories':
+        setPageLayoutMetadata({
+          ...basePageMetadata,
+          description: 'copy pending',
+          title: `${query ? `${query} | ` : ''}Stories Search`,
+          url: { pathname: '/search/stories', query: router.query },
+        });
+        break;
+      case 'images':
+        setPageLayoutMetadata({
+          ...basePageMetadata,
+          description: pageDescriptions.images,
+          title: `${query ? `${query} | ` : ''}Image Search`,
+          url: { pathname: '/search/images', query: router.query },
+        });
+        break;
+      case 'collections':
+        setPageLayoutMetadata({
+          ...basePageMetadata,
+          description: 'copy pending',
+          title: `${query ? `${query} | ` : ''}Collections Search`,
+          url: { pathname: '/search/collections', query: router.query },
+        });
+        break;
+
+      default:
+        break;
+    }
+  }, [currentSearchCategory]);
+
+  return (
+    <CataloguePageLayout {...pageLayoutMetadata}>
+      <div className="container">
+        <input placeholder="search..." type="search" />
+        <NavBar aria-label="Search Categories">
+          <NavList>
+            <NavItem>
+              <Link scroll={false} passHref href={'/search'}>
+                <NavLink
+                  aria-current={
+                    currentSearchCategory === 'overview' ? 'page' : 'false'
+                  }
+                >
+                  Overview
+                </NavLink>
+              </Link>
+            </NavItem>
+            <NavItem>
```

Comment:
Note to ourselves, we could make this a component to loop over when we remake this component